std/algorithm/sorting.d

```diff
@@ -1062,17 +1062,20 @@ sort(alias less = "a < b", SwapStrategy ss = SwapStrategy.unstable,
 @safe pure nothrow unittest
 {
     int[] array = [ 1, 2, 3, 4 ];
+
     // sort in descending order
-    sort!("a > b")(array);
+    array.sort!("a > b");
     assert(array == [ 4, 3, 2, 1 ]);
+
     // sort in ascending order
-    sort(array);
+    array.sort();
     assert(array == [ 1, 2, 3, 4 ]);
-    // sort with a delegate
+
+    // sort with a delegate and chain
     bool myComp(int x, int y) @safe pure nothrow { return x > y; }
```

Comment:
I should be more critical when looking at existing Phobos code - thanks for the idea, it looks a lot better!
Should I directly call it in the same line?

```D
assert(array.sort!((x, y) => x > y).release == [ 4, 3, 2, 1 ]);
```